install_config/router/default_haproxy_router.adoc

```diff
@@ -648,6 +648,88 @@ $ oc scale dc/router --replicas=0 && oc scale dc/router --replicas=1
 ----
 
 
+[[finding-router-hostname]]
+== Finding the Hostname of the Router
+
+When the user exposes a service, the host name in the route is the DNS name that external
+users will use to access the application. The network admin for the external network must
+make sure the host name resolves to the name of a router that has admitted the route.
+This can be done by setting up a DNS CNAME record to point to the host name of the router.
+
+The host name of the router may not be know to the user. When it is not, the cluster administrator
+will provide it.  As a convenience, The cluster admin can provide the *--router-canonical-hostname* when
+creating the router. The *--routeri-canonical-hostname* is a string that contains the external
+canonical hostname of the router.
+
+For routers that already exist, the cluster admin can edit the router's dc and add the information
+in the *ROUTER_CANONICAL_HOSTNAME* environment variable.  The current *ROUTER_iCANONICAL_HOSTNAME*
+is displayed in the route status for all routers that have admitted the route.
+
+====
+----
+# oadm router myrouter --router-canonical-hostname="rtr.example.com"
+----
+====
+
+The above command ends up creating an environemnt variable *ROUTER_CANONCAL_HOSTNAME* in the
+Router's dc. *ROUTER_CANONCAL_HOSTNAME* contains the host name of the router.
+
+The user can run *oc get route -o yaml* to show the route status. The route status
+includes a list of routers that have admitted the route. The user selects a router from the
+list and finds the host name of the router to pass along to the network admin.
```

Comment:
@knobunc Isn't router to application a potentially many to one mapping? You can pick any router that admits the route and get traffic to the application. What do you want to say here?